internal/controllers/clusterextension_controller.go

```diff
@@ -257,6 +258,10 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1.Cl
 		// Wrap the error passed to this with the resolution information until we have successfully
 		// installed since we intend for the progressing condition to replace the resolved condition
 		// and will be removing the .status.resolution field from the ClusterExtension status API
+		l := ctrl.Log.WithName("operator-controller-unpacker")
```

Comment:
Have we any reason for not using clusterextension-controller, for example, in the name? Why unpacker

Reply:
There are two unpackers, one in catalogd, one in operator-controller. Since they now share the same codebase, this differentiates the two cases a bit. This is also because the error is generated by the unpacker.